.github/config.yml

```diff
@@ -11,8 +11,7 @@ newIssueWelcomeComment: |
 # Comment to be posted to on PRs from first time contributors in your repository
 newPRWelcomeComment: |
  Thanks for opening this pull request!
- There may be some errors, **but don't worry!** We're here to help! 👍🎉😄
- @publicab/is-reviewers will be here to help you :)
+ There may be some errors, **but don't worry!** @publiclab/is-reviewers is here to help you 👍🎉😄
```

Comment:
How about is-reviewers will be here to help?

Reply:
Also add the exclamation mark. So final "is-reviewers will be here to help!(emojis)"

Reply:
Lets make it later.

Reply:
Ok